web/app/widgets/button/button.settings.tpl.html

```diff
@@ -246,6 +246,16 @@ <h3>Button Settings</h3>
                             </div>
                         </div>
                     </div>
+                    <div ng-if="form.action_type=='navigate' && form.show_item_value">
```

Comment:
If the actiontype is navigate, the controller logic needs to know when to switch background/foreground colors. This setting only applies to navigate with show item value.

Reply:
Damn, you're right :) Completely overlooked this...
Your solution was nice but there is no "inactive" value - it's simply anything different than active :)